src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static Color getRandomColor() {
+        Color[] colors = Color.values();
+        Random random = new Random();
```

Comment:
it is necessary to put the random into a class variable, an example can be seen in the checklist

Reply:
not fixed